jcommune-plugins/questions-n-answers-plugin/src/main/resources/org/jtalks/jcommune/plugin/questionsandanswers/template/question.vm

```diff
@@ -180,7 +180,7 @@ Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
         <div>
           <div class="question-left-panel pull-left">
             <div class="thumbnail wraptocenter">
-              <img src="${request.contextPath}/users/${post.userCreated.id}/avatar" alt=""/>
+              <img src="${request.contextPath}/users/${post.userCreated.id}/avatar?version=${post.userCreated.version}" alt=""/>
```

Comment:
I can't know. This part of code is not mine, it was being here before I changed the line. My changes only affect version of avatar.